TypeCobol.Analysis/Cfg/ControlFlowGraphBuilder.cs

```diff
@@ -1279,7 +1284,7 @@ private void ComputeBasicBlockGroupTerminalBlocks(BasicBlockForNodeGroup group)
             if (group.Group.Count > 0 && group.TerminalBlocks == null)
             {
                 LinkedListNode<BasicBlock<Node, D>> first = group.Group.First;
-                MultiBranchContext ctx = new MultiBranchContext(null);
+                MultiBranchContext<Node, D> ctx = new MultiBranchContext<Node, D>(null);
```

Comment:
So this must be done in another issue, because TypeCobol_Qualis will also be concerned. It should not be done in this PR and there are other classes that should also be changed if so. It is not an urgent task.

Reply:
Ok it was already noted in #1783.